tags.json

```diff
@@ -1313,6 +1313,7 @@
     "authentication",
     "secure"
   ],
+  "keyboard": [],
```

Comment:
```suggestion
  "keyboard": [
    "layout"
    "spell",
    "settings",
    "mouse"
  ],
```

Reply:
thanks, I couldn't think ideas for tags earlier